### ISSUE TITLE
Disk space after export

### ISSUE THREAD

**Issue by Philippe-M** [akeneo/pim-community-dev#13077]
Hello,

I often encounter a disk space problem. The cause is the generation of exports. The files will be stored in the /var/www/html/pim-community-standard/app/archive/export/folder. In my case, we often use the export in Excel so it's in the xlsx_product_export folder. By analyzing I realized that the export job creates an Excel file but copies all the files of each product and then creates a zip archive. Once the job is finished, everything remains as it is, so the space occupied is much more important than the reality:

```
/var/www/html/pim-community-standard/app/archive/export/xlsx_product_export# ll
total 4
drwxr-xr-x 5 www-data www-data 4096 oct.  22 11:59 806

/var/www/html/pim-community-standard/app/archive/export/xlsx_product_export# du -hs 806
2,3G	806

/var/www/html/pim-community-standard/app/archive/export/xlsx_product_export/806# ll
total 12
drwxr-xr-x 2 www-data www-data 4096 oct.  22 11:59 archive
drwxr-xr-x 2 www-data www-data 4096 oct.  22 11:59 log
drwxr-xr-x 3 www-data www-data 4096 oct.  22 11:58 output

/var/www/html/pim-community-standard/app/archive/export/xlsx_product_export/806# du -hs archive/
1,1G	archive/
/var/www/html/pim-community-standard/app/archive/export/xlsx_product_export/806# du -hs output/
1,2G	output/

/var/www/html/pim-community-standard/app/archive/export/xlsx_product_export/806# ll archive/
total 1124772
-rw-r--r-- 1 www-data www-data 1151759488 oct.  22 11:59 export_export_ordre_fiche_article_2020-10-22_09-57-39.zip

/var/www/html/pim-community-standard/app/archive/export/xlsx_product_export/806# ll output/
total 28
-rw-r--r--  1 www-data www-data 23832 oct.  22 11:58 export_export_ordre_fiche_article_2020-10-22_09-57-39.xlsx
drwxr-xr-x 44 www-data www-data  4096 oct.  22 11:58 files

/var/www/html/pim-community-standard/app/archive/export/xlsx_product_export/806# du -hs output/files/
1,2G	output/files/
```

The output / files directory contains the sub-folders of each product and each sub-folder contains the various media linked to a product (images, pdf, office, etc.).

Why is this folder not deleted after the zip archive is generated?

**Comment by halohunter** [comment]
The files get deleted by CHRON job after 30 days if I remember correctly. You could shorten this to as little as 15 minutes which would resolve your disc space problem.

You can also configure an export job to not create a zip file with the asset data.
